src/bundle/Request/ParamConverter/SuggestionQueryParamConverter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Search\Request\ParamConverter;
+
+use Ibexa\Search\Model\SuggestionQuery;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+final class SuggestionQueryParamConverter implements ParamConverterInterface
```

Comment:
Are we switching from ParamConverters to Symfony Attributes? @alongosz

Reply:
> Are we switching from ParamConverters to Symfony Attributes? @alongosz

We should avoid using `FrameworkExtraBundle` because it has been abandoned. @mikadamczyk  Not sure WDYM about attributes. There's Symfony Argument Resolver for that, e.g: https://github.com/ibexa/shipping/tree/main/src/bundle/ControllerArgumentResolver

Reply:
that is what I was thinking, thanks @alongosz